### PR TITLE
Add a butchering kit made of butcher knife, carving knife and meat cleaver

### PR DESCRIPTION
**Pull request by ampersand55**
#### Summary

```SUMMARY: Content "Add a butchering kit made of butcher knife, carving knife and meat cleaver"```

#### Purpose of change

You would be able to butcher more efficiently if you had access to specialized tools for each part of the butchering process. 

#### Describe the solution

json edits.

Increased butchering quality bonus (~~33~~ ~~40~~ 37) if you have butchering knife, carving knife and meat cleaver assembled in a kit. 

For reference, butchering knife and meat cleaver have a butchering quality of 25, hunting knife 30 and autonomous surgical scalpels 50.

Volume, weight and price the same as the other knifes combined. 

Damage is from the highest damage knife in the set (meat cleaver).

#### Describe alternatives you've considered

Making a skinning knife and adding it to the kit.

#### Testing

Tested the recipe in game.

#### Additional context

UPDATES:

I've incorporated suggestions. A basic package 2 rags and 2 strings for carrying the butchering kit, as a common way of transporting knives is wrapping them in a piece of cloth. See: [1](https://cnz.to/tools-utensils/how-to-transport-your-knives/), [2](https://www.strongmove.co.uk/blog/how-to-pack-knives-for-moving/).

Also lowered the butchering quality slightly to 37, which is a 50% increase from basic butchering knife.

Moved the recipe from kitchen_knives.json to tools.json.

Increased weight to match 2 rags used for wrapping.

 screenshot

### CODE REVIEW

**Comment by Brian-Otten** [comment]
I would add the hacksaw (or add a bonesaw to the game in some suitable spots and use that), and buff the butchering value a bit higher. at 33 it feels a bit weak compared to hunting knife while you are now much better equipped, 40 seems like a good value to me.

**Comment by ampersand55** [comment]
Interesting idea. However, a saw would only be used for larger mammals. You wouldn't need it for butchering e.g. invertebrates, squirrels or fish. But then again, you would probably not need all of these tools for everything either.

Who thinks we should add hacksaw or bonesaw? I'm going to go with popular vote. Other feedback is of course also welcome.

**Comment by Brian-Otten** [comment]
Popular vote is not really done because it's finicky and slow and has it's own issues, so use whatever you think is best and you can convince others of. Your previous version is also fine, but i feel at 33 butchering skill it will probably go underutilized.

**Comment by ampersand55** [comment]
You're probably right. Increasing butchering quality to 40.

**Comment by LaVeyanFiend** [comment]
Shouldn't they be encased in something to keep them together?

File: data/json/recipes/other/tool.json [hunk 1]
```diff
@@ -146,6 +146,17 @@
...
     "tools": [ [ [ "swage", -1 ] ] ]
   },
+  {
+    "result": "knife_butchering_kit",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+	"reversible": true,
+    "time": "1 s",
```
Comment (LaVeyanFiend) [inline data/json/recipes/other/tool.json:156]:
```suggestion
	  "reversible": true,
    "time": "1 s",
```

Comment (LaVeyanFiend) [inline data/json/recipes/other/tool.json:156]:
```suggestion
    "skill_used": "fabrication",
	  "reversible": true,
    "time": "1 s",
```
Should have equal indent

**Comment by Pupsi-Mupsi** [comment]
Please lint the jsons.

http://dev.narc.ro/cataclysm/format.html

**Comment by Brian-Otten** [comment]
I think having some kind of "box" included would limit the crafting of the item in a weird way while paradoxically making the set harder to store. If we want to double down on keeping them together as a set you could could add a generic cordage requirement to imply your character ties the bundle together when not in use.

**Comment by Pupsi-Mupsi** [comment]
I like the idea very much!

Would you consider:

Isn't from a balance perspective 40 butchering quality and 1 second crafting time with no skills required using 3 quite common components a bit too much.

Hunting knife and better butchering tools  are pretty rare.
What do you think?

**Comment by FrozenSnowFox** [comment]
> I think having some kind of "box" included would limit the crafting of the item in a weird way while paradoxically making the set harder to store. If we want to double down on keeping them together as a set you could could add a generic cordage requirement to imply your character ties the bundle together when not in use.

From what I've seen knife sets usually use a material case of some kind.  Some even come with waist or shoulder straps.  Something like that could probably be made with tailoring or even using an existing small backpack.

**Comment by ampersand55** [comment]
I've incorporated suggestions. A basic package 2 rags and 2 strings for carrying the butchering kit, as a common way of transporting knives is wrapping them in a piece of cloth. 

Also lowered the butchering quality slightly to 37, which is a 50% increase from basic butchering knife.

Moved the recipe from kitchen_knives.json to tools.json. 

screenshot

**Comment by Shodan14** [comment]
A dissection kit would be a nice addition once we get more random surgical tools, you can't really do that much with just a scalpel.

File: data/json/items/tool/cooking.json [hunk 2]
```diff
@@ -17,6 +17,19 @@
     "//": "It would be nice if this could have a use action to thaw frozen stuff.  It's not hot enough to work as a hotplate.",
     "flags": [ "LEAK_DAM", "RADIOACTIVE", "DURABLE_MELEE" ]
   },
+  {
+    "id": "butchering_kit",
+    "name": { "str": "butchering kit", "str_pl": "butchering kits" },
+    "type": "TOOL",
+    "description": "This is a kit with all the specialized knifes you would need for deboning, skinning and preparing the cuts of meat from an animal.",
```
Comment (aoae23) [inline data/json/items/tool/cooking.json:24]: Maybe reword to "This is a kit containing all the specialized knives you need to debone, skin, and prepare cuts of meat from an animal."

Comment (ampersand55) [reply data/json/items/tool/cooking.json]: Yea, that's better.
